### PR TITLE
18-loading-icon

### LINKED ISSUE #18: Loading icon...
- Page shows a loading icon until the data has been received from the api.

### PR DESCRIPTION
**Pull request by itggot-kianoosh-geravesh**
- Created and added a loading icon which shows before data is received from the APIs (resolved #18)
- Moved tweets to a new separated component file 
- Removed faCoffee made during testing
- Removed extra importing in the other files

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -4,6 +4,8 @@
 </template>
 
 <style lang="sass">
-  #app
+  #app, html
```
Comment (itggot-linus-styren) [inline src/App.vue]: Do we need to apply this rule to both `#app` and `html`?

File: src/components/LoadingIcon.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<template lang="pug">
+    b-jumbotron
+      div#loading.mx-auto
+        div(v-for="x in 8")
```
Comment (itggot-linus-styren) [inline src/components/LoadingIcon.vue]: What are these divs used for?

File: src/components/TweetList.vue [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<template lang="pug">
+    b-jumbotron
+        b-list-group
+            b-list-group-item(v-for="(node, index) in tweets" :key="index").flex-column.align.items.start
+                .d-flex.w-100.justify-content-between
+                h5.mb-1 {{node['user']}}
+                small 3 days ago
```
Comment (itggot-linus-styren) [inline src/components/TweetList.vue]: This was changed in fe8da89dbb31d886f2f3909ab113633c5f348d09 and should be updated here.

File: src/views/Tweets.vue [hunk 4]
```diff
@@ -1,33 +1,35 @@
...
 })
+
 export default class Tweets extends Vue {
   twitter_endpoint : string = `https://us-central1-saot-217513.cloudfunctions.net/tweets`;
   tweets : Array<Object> = [];
+  done : boolean = false;
 
   created () {
-    this.search(this.$route.params.query);
+    if (!this.$route.params.query) {
```
Comment (itggot-linus-styren) [inline src/views/Tweets.vue]: Consider saving this to a temporary variable like `let query = this.$route.params.query;` to avoid duplication.

Comment (itggot-kianoosh-geravesh) [reply src/views/Tweets.vue]: - fixed fd74b218ad0d36adbc9ab08a663495dee5a2fb62

File: src/components/TweetList.vue (hunk 3 above)
Comment (itggot-kianoosh-geravesh) [reply src/components/TweetList.vue]: - fixed fd74b218ad0d36adbc9ab08a663495dee5a2fb62

**Pull request merged**